lib/analysis/community.py

```diff
@@ -22,7 +22,6 @@ def infomap_igraph(ig_graph, net_file_location=None):
 
```

Comment:
The documentation for args need to be updated. The reduce_graph(bool) argument should be removed.

Reply:
done